### ISSUE TITLE
Is opal docker using proxy settings?

### ISSUE THREAD

**Issue by fnwirth** [obiba/opal#3688]
I created an opal installation based on the current opal docker image. On my local machine everthing works fine. When I start the installation on a server in a data chenter I get the log entries listed below. The server is behind a proxy, however, I configured the proxy in docker and also in the OS. I am e.g. able to execute `wget https://plugins.obiba.org` successfully. Is there a specific proxy configuration necessary?
This is possible a duplicate to #3687

```
2021-12-01 11:11:43,002 [main] INFO  org.obiba.opal.core.install.DefaultPluginsInstallStep - Installing default plugins from update site: https://plugins.obiba.org
2021-12-01 11:11:43,002 [main] INFO  org.obiba.opal.core.install.DefaultPluginsInstallStep - Installing opal-search-es plugin
2021-12-01 11:13:53,353 [main] ERROR org.obiba.opal.core.install.DefaultPluginsInstallStep - Failed to install opal-search-es plugin
org.obiba.plugins.PluginRepositoryException: Cannot update plugin site: Connection timed out (Connection timed out)
        at org.obiba.plugins.PluginRepositoryCache.initializePluginRepository(PluginRepositoryCache.java:78)
        at org.obiba.plugins.PluginRepositoryCache.getOrUpdatePluginRepository(PluginRepositoryCache.java:47)
        at org.obiba.plugins.PluginRepositoryCache.getPluginLatestVersion(PluginRepositoryCache.java:98)
        at org.obiba.opal.core.service.PluginsServiceImpl.installPlugin(PluginsServiceImpl.java:132)
        at org.obiba.opal.core.install.DefaultPluginsInstallStep.execute(DefaultPluginsInstallStep.java:39)
        at org.obiba.runtime.upgrade.support.DefaultUpgradeManager.executeNewInstallation(DefaultUpgradeManager.java:151)
        at org.obiba.runtime.upgrade.support.DefaultUpgradeManager.executeUpgrade(DefaultUpgradeManager.java:117)
        at org.obiba.opal.server.UpgradeCommand.standardUpgrade(UpgradeCommand.java:63)
        at org.obiba.opal.server.UpgradeCommand.execute(UpgradeCommand.java:57)
        at org.obiba.opal.server.OpalServer.upgrade(OpalServer.java:68)
        at org.obiba.opal.server.OpalServer.<init>(OpalServer.java:32)
        at org.obiba.opal.server.OpalServer.main(OpalServer.java:117)
Caused by: java.net.ConnectException: Connection timed out (Connection timed out)
        at java.net.PlainSocketImpl.socketConnect(Native Method)
        at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
        at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
        at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
        at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
        at java.net.Socket.connect(Socket.java:607)
        at sun.security.ssl.SSLSocketImpl.connect(SSLSocketImpl.java:288)
        at sun.security.ssl.BaseSSLSocketImpl.connect(BaseSSLSocketImpl.java:173)
        at sun.net.NetworkClient.doConnect(NetworkClient.java:180)
        at sun.net.www.http.HttpClient.openServer(HttpClient.java:463)
        at sun.net.www.http.HttpClient.openServer(HttpClient.java:558)
        at sun.net.www.protocol.https.HttpsClient.<init>(HttpsClient.java:264)
        at sun.net.www.protocol.https.HttpsClient.New(HttpsClient.java:367)
        at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.getNewHttpClient(AbstractDelegateHttpsURLConnection.java:203)
        at sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1162)
        at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:1056)
        at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:189)
        at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1570)
        at sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1498)
        at sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:268)
        at java.net.URL.openStream(URL.java:1068)
        at com.fasterxml.jackson.core.TokenStreamFactory._optimizedStreamFromURL(TokenStreamFactory.java:185)
        at com.fasterxml.jackson.core.JsonFactory.createParser(JsonFactory.java:944)
        at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3376)
        at org.obiba.plugins.PluginRepositoryCache.initializePluginRepository(PluginRepositoryCache.java:74)
        ... 11 common frames omitted
2
```

**Comment by ymarcon** [comment]
The official opal docker image already includes all the plugins, so this error should not be a blocker for a functional opal server.
Yes, it is a duplicate of #3687

**Comment by fnwirth** [comment]
Thanks for your answer.
My problem is that subsequently to this error, changes that I perform when administering the opal server are not administered.
For instance, I create the mongodb database connections manually since they are not created automatically subsequently to the update error. I can create and test the connections. However, after restarting with docker-compose the created connections are gone. Same applies to the usage of an own https certificate. 
This happens as far as I can tell only after the update plugins error. On another machine without proxy everything works fine.
